src/components/shared/thread/StackGraph.js

```diff
@@ -177,7 +181,7 @@ class StackGraph extends PureComponent<Props> {
       ctx.fillStyle = color;
       for (let i = 0; i < samplesBucket.height.length; i++) {
         const height = samplesBucket.height[i];
-        const startY = canvas.height - height;
+        const startY = upsideDown ? 0 : canvas.height - height;
```

Comment:
Who you callin upsideDown here? It's you who is upside down!

Reply:
Just kidding, but maybe saying which way right side up is, or maybe renaming it to "stacksGrowFromCeiling" or "stacksGrowFromFloor" or something similar might help.